sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java

```diff
@@ -165,10 +165,10 @@ public interface BigQueryOptions
   void setStorageWriteApiMaxRequestSize(Long value);
 
   @Description(
-      "If set, BigQueryIO.Read will use the StreamBundle based"
-          + "implementation of the Read API Source")
+      "If set, BigQueryIO.Read will rely on the Read API backends to surface the appropriate"
+          + " number of streams for read")
   @Default.Boolean(false)
-  Boolean getEnableBundling();
+  Boolean getEnableStorageReadApiV2();
 
-  void setEnableBundling(Boolean value);
+  void setEnableStorageReadApiV2(Boolean value);
```

Comment:
Should this option be removed entirely?

Reply:
The effect of this option is to set the number of stream requested to 0 so the server will decide an appropriate number of read streams (which is Read API source v2). It is preserved for now.

Reply:
I see, makes sense

Reply:
Nit: Could we rename this to `enableStorageReadApiSourceV2`? 

There are currently multiple versions of the Read API and the current name could possibly be confusing as to which version of the API is being used.